public/wp-content/themes/glific/page-get-started.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Get Started page for the theme
+ *
+ * @package coloredcow
+ */
+get_header();
+
+?>
+
+<div class="d-flex flex-column main-container bg-theme-white-smoke">
+    <div class="justify-content-between w-235 w-md-310 mx-auto pt-18 pt-md-18 pb-md-13.5 pb-11.5">
+        <h1 class="font-heebo-bold fz-28 leading-40 fz-md-36 text-theme-primary"><?php echo get_field('main_heading'); ?></h1>
+    </div>
+    <div class="subsection-container d-flex flex-column ml-6 ml-xl-26 w-xl-936 w-md-712">
+        <?php $section_one_data = get_field('section_one');
+            if (!empty($section_one_data)) :
+                foreach ($section_one_data as $key => $data) : ?>
+                <div class="flex-column flex-md-row d-flex justify-content-between mb-13.5 mb-xl-18 mb-md-20 w-full">
+                    <div class="d-flex flex-row w-xl-67">
+                        <h2 class="font-heebo-light fz-60 text-theme-mine-shaft">0<?php echo $key+1; ?></h2>
+                    </div>
+                    <div class="d-flex flex-column w-280 w-xl-770 w-md-613 rounded-30 bg-theme-gin get-started-content-box">
+                        <div class="bg-white p-6 pt-md-6 pb-md-4.5 rounded-20 pl-md-8.5">
+                            <h2 class="fz-24 font-heebo-regular text-theme-mine-shaft mb-0"><?php echo $data['title']; ?></h2>
+                        </div>
+                        <div class="d-flex flex-column bg-theme-gin rounded-bottom-30 font-heebo-regular fz-18 leading-28 text-theme-mine-shaft pt-8 pb-4 pl-5 pb-md-6 pl-md-8">
+                        <?php if (!empty($data['content'])) : 
+                            foreach ($data['content'] as $index => $content) : ?>
```

Comment:
Why do we need `$index` here? I can't see you are using it anywhere.
```suggestion
                            foreach ($data['content'] as $content) : ?>
```